File: C99/1065.c
#include <stdio.h>
 
int main() {
 
    int a,r;
    
    for(int i=0;i <5;i++){
        scanf("%i",&a);
        if(a%2 == 0){
            r++;
        }
    }
    printf("%i valores pares\n",r);
    return 0;
}

File: Java/1065.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {

    public static void main(String[] args) throws IOException {
        
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        int n, i, pares = 0;
        
        for(i = 0; i < 5; i++) {
            n = Integer.parseInt(in.readLine());
            if(n % 2 == 0) {
                pares++;
            }
        }
        System.out.println(pares + " valores pares");
    }

}